t/consistent-hashed-upstream.t

```diff
@@ -0,0 +1,90 @@
+# vi:filetype=
+
+use lib 'lib';
+use Test::Nginx::Socket;
+
+#repeat_each(3);
```

Comment:
Any reason for not enabling `repeat_each`? Is it a leftover of local debugging or a thing done on purpose due to the nature of the tests?